app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
[OPTIONAL] if these codes are necessary you can remove them as well. 😉

Reply:
@Rustamxon7,  all have been deleted